crates/bevy_log/src/lib.rs

```diff
@@ -115,7 +115,7 @@ impl Plugin for LogPlugin {
         }
 
         let default_filter = { format!("{},{}", self.level, self.filter) };
-        LogTracer::init().unwrap();
+        LogTracer::init().expect("Could not set global logger as it has already been set. Please disable LogPlugin from Bevy's DefaultPlugins");
```

Comment:
Wonder if we should just downgrade this to a warning instead
```suggestion
        LogTracer::init()
                  .unwrap_or_else(|| warn!("Could not set global logger as it has already been set. Please disable LogPlugin from Bevy's DefaultPlugins"));
```

Reply:
Yeah, I think that's pretty reasonable.